docs/source/eventing/actions/label.rst

```diff
@@ -20,10 +20,10 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | TO       | |yes|            | |no|                | New text                                          |
 +----------+------------------+---------------------+---------------------------------------------------+
-| COLOR    | |no|             | |no|                | New color. If omitted, color is set to black"     |
-+----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
+| COLOR    | |no|             | |no|                | New color. If omitted, color is set to black"     |
++----------+------------------+---------------------+---------------------------------------------------+
```

Comment:
Now the new PR for the typo (which i will merge first).

Reply:
New PR to fix typo done.

I will remove the quotes also here, to be "in sync" **after** merging the new PR

Reply:
You should be in sync with that PR because i will merge that first.